<launch>
    <node pkg="tf" type="static_transform_publisher" name="world2odom"
    args="0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 1.0000 odom orb_slam 100"/>

<!--     <node pkg="ORB_DVL2" type="cameras_black_node.py" name="camera_black" output="screen"/> -->
<!--     <node pkg="ORB_DVL2" type="align_EKF.py" name="ekf_align" output="screen"/> -->
<!--    <node pkg="ORB_DVL2" type="camera_info_pub.py" name="camera_info_puber"/>-->
<!--    <group ns="camera">-->
<!--        <node pkg="stereo_image_proc" type="stereo_image_proc" name="stereo_image_proc"-->
<!--              args="_approximate_sync:=true">-->
<!--            <param name="queue_size" value="2000" />-->
<!--            <param name="approximate_sync" value="True" />-->
<!--            <param name="correlation_window_size" value="21" />-->
<!--            <param name="min_disparity" value="0" />-->
<!--            <param name="disparity_range" value="128" />-->
<!--            <param name="uniqueness_ratio" value="30" />-->
<!--            <param name="texture_threshold" value="100" />-->
<!--            <param name="speckle_size" value="1000" />-->
<!--            <param name="speckle_range" value="1" />-->
<!--            <param name="stereo_algorithm" value="1" />-->
<!--        </node>-->
<!--    </group>-->
    <param name="/dense_mapper/out_path" value="/home/da/project/frontier/docker_devel_slam/auip_ws/src/AUIP_slam/dense"/>
<!--    <node pkg="ORB_DVL2" type="dense_mapping_node" name="dense_mapping_node" output="log"/>-->
<!--    <node pkg="ORB_DVL2" type="pose_repub" name="pose_repub_node" output="log"/>-->
    <node pkg="ORB_DVL2" type="zed2i_rectification_node.py" name="'zed2i_image_rect_node"/>
<!--    <node pkg="ORB_DVL2" type="dvl_twist" name="dvl_twist_pub" output="log"/>-->
<!--    <node pkg="ORB_DVL2" type="bluerov2_dvl_repub.py" name="dvl_twist_pub" output="log"/>-->

    <node pkg="ORB_DVL2" type="stereo_dvl2" name="stereo_dvl"
          args="$(find ORB_DVL2)/Vocabulary/ORBvoc.txt $(find ORB_DVL2)/data/underwater_orbslam3_blue_zed2i.yaml false" output="screen"/>
<!--     <node pkg="ORB_DVL2" type="stereo_dvl2" name="stereo_dvl"-->
<!--               args="$(find ORB_DVL2)/Vocabulary/ORBvoc.txt $(find ORB_DVL2)/data/underwater_orbslam3_falcon.yaml false" output="screen"/>-->
<!--    <node pkg="ORB_DVL2" type="BlueOdom2Twist.py" name="BlueOdom2Twist" />-->
    <arg name="model" default="$(find ORB_DVL2)/urdf/bluerov.urdf"/>
    <param name="robot_description" command="$(find xacro)/xacro $(arg model)"/>
    <param name="/ORBSLAM3_tightly/out_path" value="/home/da/project/ros/orb_dvl2_ws/src/dvl2/orb3_result/"/>
    <param name="/ORBSLAM3_tightly/traj_path" value="/home/da/project/ros/orb_dvl2_ws/src/dvl2/dvl2_results/stamped_traj_estimate0.txt"/>
    <param name="/ORBSLAM3_tightly/map_file" value="/home/da/project/ros/orb_dvl2_ws/src/dvl2/orb3_result/Altlas.osa"/>
    <param name="/ORBSLAM3_tightly/is_load_map" value="false" type="bool"/>
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find ORB_DVL2)/launch/falcon_tightly.rviz" output="log"/>
<!--    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find ORB_DVL2)/launch/falcon_tightly.rviz" launch-prefix="bash -c 'exec rviz \'$@\' >/dev/null 2>&1' &#45;&#45;" output="screen"/>-->

</launch>